eng/Signing.props

```
@@ -19,4 +19,8 @@
     <FileExtensionSignInfo Update=".js" CertificateName="None" />
     <FileExtensionSignInfo Include=".vbs" CertificateName="None" />
   </ItemGroup>
+  
+  <ItemGroup>
+    <ItemsToSignPostBuild Include="$(ArtifactsDir)VSSetup\$(Configuration)\Insertion\**\*.msi" />
```

I think there's an arcade property for the path we can use (https://github.com/dotnet/arcade/blob/e7ede87875f41a9b3df898ae08da5ebc96e24f56/src/Microsoft.DotNet.Arcade.Sdk/tools/RepoLayout.props#L62)

```suggestion
    <ItemsToSignPostBuild Include="$(VisualStudioSetupInsertionPath)**\*.msi" />
```